### PR TITLE
Campaigns page redesign

### LINKED ISSUE #34: Campaigns page needs redesign
The page showing all the campaigns ("/campaigns") needs to be updated to match the v3 designs.

### LINKED ISSUE #95: Past Campaigns on the website
Description:
**As a:** user
**Currently**: I cannot see the past campaigns on the website
**I want:** to be able to see the past campaigns on the website
**Because:** so that I can view past content that now-u has done and view the actions

Reporter:
James Elgar 

Trello link:
https://trello.com/c/JDmNX8lx/230-past-campaigns-on-the-website

Comments:

### PR DESCRIPTION
**Pull request by stellenberger**
# Description

Please include a brief summary of what the issue is and what has been done to solve it?

This is the V12C designs for campaigns page. 

Fixes #34 
Fixes #95

# Checklist:

- [🌶️ ] I have commented my code, particularly in hard-to-understand areas
- [🌶️ ] I have reviewed the 'files changed' tab on github to ensure all changes are expected
- [🌶️ ] I have added someone to review the pr

### CODE REVIEW

File: src/components/app_download/AppDownload.module.scss [hunk 1]
```diff
@@ -60,13 +60,18 @@
...
-  @media screen and (max-width: 500px) {
+  @media screen and (max-width: $medium) {
     width: 100%;
   }
   button {
     width: 40%;
     max-width: 170px;
     margin: 5px;
+    @media screen and (max-width: $medium) {
+      // width: 100%;
```
Comment (JElgar) [inline src/components/app_download/AppDownload.module.scss:71]:
```suggestion
```

File: src/components/campaign_poster/CampaignPoster.module.scss [hunk 2]
```diff
@@ -2,35 +2,57 @@
...
+  box-shadow: 0 0 10px 0px rgba($color: #000000, $alpha: 0.08);
+  @media (max-width: $medium) {
+    flex-wrap: wrap;
+    box-shadow: none;
   }
-  & h4 {
-    line-height: 1;
+  transition: 0.5s;
+  &:hover {
+    transform: scale(1.01);
```
Comment (JElgar) [inline src/components/campaign_poster/CampaignPoster.module.scss:14]: 😍

File: src/domain/campaigns/Campaigns.jsx [hunk 3]
```diff
@@ -1,25 +1,37 @@
...
+import { CampaignsGroup, AppDownloadButtons } from '../../components';
+import { campaignsURL, pastCampaignsURL } from '../../utils/constants';
 import classes from './Campaigns.module.scss';
 
 const Campaigns = () => {
   const [campaigns, setCampaigns] = useState([]);
-  const [upcomingCampaigns] = useState(null);
-
+  const [pastCampaigns, setPastCampaigns] = useState([]);
+  const Months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```
Comment (JElgar) [inline src/domain/campaigns/Campaigns.jsx:9]: should we shove this in `constant.js`

**Review by JElgar — APPROVED** [review]
Few comments but other than that love it!

File: src/domain/campaigns/Campaigns.jsx (hunk 3 above)
Comment (stellenberger) [reply src/domain/campaigns/Campaigns.jsx:9]: Yes absolutely!

**Comment by tunangst** [comment]
Extremely well done

File: src/components/campaign_poster/CampaignPoster.module.scss [hunk 4]
```diff
@@ -2,35 +2,57 @@
...
+    overflow: hidden; // edge case placeholder
+    @media (max-width: $medium) {
+      max-height: 200px;
+      height: auto;
+      width: 100%;
+    }
+    img {
+      width: 100%;
+      height: auto;
+    }
```
Comment (tunangst) [inline src/components/campaign_poster/CampaignPoster.module.scss:30]: Not sure if you want to go this direction, but some images fit a little better out of our defined size variables.
`img {`
`   width: 100%;`
`   height: 100%;`
`   object-fit: cover`
`}`

Comment (JElgar) [reply src/components/campaign_poster/CampaignPoster.module.scss:30]: Yep thats a good shout! Btw @tunangst you can give suggestions with one of the buttons (The one next to header text) and that means they can just be committed directly. 

eg
```suggestion
    img {
      width: 100%;
      height: auto;
      object-fit: cover;
    }
```

Comment (tunangst) [inline src/components/campaign_poster/CampaignPoster.module.scss:30]:
```suggestion
    img {
      width: 100%;
      width: 100%;
      object-fit: cover;
    }
```

Comment (tunangst) [reply src/components/campaign_poster/CampaignPoster.module.scss:30]: ohh I see, thank you :]

Comment (stellenberger) [reply src/components/campaign_poster/CampaignPoster.module.scss:30]: Yes great idea - thanks @tunangst !

**Pull request merged**